pkg/console/starter/starter.go

```diff
@@ -220,9 +220,11 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		return err
 	}
 
+	contentSecurityPolicyEnabled := featureGates.Enabled("ConsolePluginContentSecurityPolicy")
```

Comment:
Hey, @JoelSpeed. Would you mind confirming this is right for content security policy? We're disabling some behavior in console if the gate is not enabled.

https://github.com/openshift/api/pull/2042/files#diff-d47d7d4b42f1c65f84b1a2ab7b4a1aefc599dec53bf7ab22e9d14936178ad95dR92

I wish we had named this `ConsoleContentSecurityPolicy` rather than `ConsolePluginContentSecurityPolicy`, but I assume it's too late for that (at least for 4.18?).

Reply:
That will give you what you need to gate the functionality yes

And yep, a bit late to change the name unfortunately